Sources/SwiftDocC/Model/DocumentationNode.swift

```diff
@@ -426,10 +426,10 @@ public struct DocumentationNode {
         let symbolAllowedValues = symbol![mixin: SymbolGraph.Symbol.AllowedValues.self]
         
         if let possibleValues = markupModel.discussionTags?.possiblePropertyListValues, !possibleValues.isEmpty {
-            let validator = PropertyListPossibleValuesSection.Validator(diagnosticEngine: engine)
             guard let symbolAllowedValues else {
-                possibleValues.forEach { 
-                    engine.emit(validator.makeExtraPossibleValueProblem($0, knownPossibleValues: [], symbolName: self.name.plainText))
+                let validator = PropertyListPossibleValuesSection.Validator()
```

Comment:
I'm getting an error building this branch because moving `validator` to have a smaller scope here means that the use of `validator` [on line 455](https://github.com/swiftlang/swift-docc/pull/1033/files#diff-3b01bb4d715b0180b509c021da2791a2beacf65919fbdae4d323bbcfa4a433d1R455) cannot be found because it's no longer in scope.

Reply:
Fixed in https://github.com/swiftlang/swift-docc/pull/1033/commits/ed3a95cb9631ae6840587e9f750353f9602401e6